Snowdog_Menu/templates/main-mobile/menu/node_type/product.phtml

```diff
@@ -0,0 +1,30 @@
+<?php /** @var \Snowdog\Menu\Block\NodeType\Product $block */ ?>
+<?php
+$nodeId = $block->getId();
+$class = $block->getMenuClass();
+$productUrl = $block->getProductUrl($nodeId);
+$productPrice = $block->getProductPrice($nodeId);
+$formattedProductPrice = $block->getFormattedProductPrice($nodeId);
+$productImage = $block->getProductImage($nodeId);
+$productTitle = $block->getProductTitle($nodeId);
+$productUrl = $productUrl ?: '#';
+$dataAttribute = '';
+$classLink = $block->getIsRoot() ? $class . '__label link--invert' : $class . '__label';
+$title = $block->getTitle();
+
+if ($nodeId) {
+    $dataAttribute = ' data-menu="menu-' . $nodeId . '"';
+}
+
+if ($block->isCurrentProduct($nodeId)) {
+    $classLink .= ' current';
+}
+?>
+
+<a
+    href="<?= $productUrl ?>"
```

Comment:
Could you add `<?= $block->escapeUrl($productUrl) ?>`?

Reply:
sure, added